api_compatibility_policy.md

```diff
@@ -29,35 +25,96 @@ A backwards incompatible change would be a change that requires a user to update
 
-Some of our CRDs and features are considered alpha, some are beta, and we are working
-toward GA.
+Within a stable CRD, certain opt-in features or API fields gated may be considered `alpha` or `beta`. Similarly, within a beta CRD, certain opt-in features may be considered `alpha`. See the section on Feature Gates for details.
 
 The following CRDs are considered beta, though features may be introduced that are
 alpha:
 
-* `Task`
-* `TaskRun`
-* `ClusterTask`
-* `Pipeline`
-* `PipelineRun`
-
-We are following [the Kubernetes definitions of these stages](https://kubernetes.io/docs/reference/using-api/api-overview/#api-versioning)
-and we are following [the Kubernetes deprecation policy](https://kubernetes.io/docs/reference/using-api/deprecation-policy/).
-
-* Alpha:
-  * These features may be dropped at any time, though you will be given at least
-    one release worth of warning.
-* Beta:
-  * These features will not be dropped, though the details may change.
-  * Any [backwards incompatible changes](#backwards-incompatible-changes) must be
-    introduced in a backwards compatible manner first, with a deprecation warning
-    in the release notes and migration instructions.
-  * You will be given at least 9 months to migrate before a backward incompatible
-    change is made. This means an older beta API version will continue to be
-    supported in new releases for a period of at least 9 months from the time a
-    newer version is made available.
+- `Task`
+- `TaskRun`
+- `ClusterTask`
+- `Pipeline`
+- `PipelineRun`
+
+### Alpha CRDs
+
+- For Alpha CRDs, the `apiVersion` contains the `alpha` (e.g. `v1alpha1`)
+  
+- Features may be dropped at any time, though you will be given at least one release worth of warning.
+
+### Beta CRDs
+
+- The `apiVersion` field of the CRD contains contain `beta` (for example, `v1beta1`).
+
+- These features will not be dropped, though the details may change.
+
+- Any [backwards incompatible changes](#backwards-incompatible-changes) must be introduced in a backwards compatible manner first, with a   deprecation warning in the release notes and migration instructions.
+
+- Users will be given at least 9 months to migrate before a backward incompatible change is made. This means an older beta API version will continue to be supported in new releases for a period of at least 9 months from the time a newer version is made available.
+
+- Beta API versions follow the [Go libraries compatibility policy](#go-libraries-compatibility-policy).
+
+### GA CRDs
+
+- The `apiVersion` field of the CRD is `vX` where `X` is an integer.
+
+- Stable API versions remain available for all future releases within a Tekton major version.
```

Comment:
does "tekton major version" here refer to semver?

Reply:
yes - will update the wording to include semver major version